appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/lifecycle/ChildNodeLifecycleManager.kt

```diff
@@ -25,7 +23,7 @@ internal class ChildNodeLifecycleManager<InteractionTarget: Any>(
     private val coroutineScope: CoroutineScope,
 ) {
 
-    private val lifecycleState = MutableStateFlow(Lifecycle.State.INITIALIZED)
+    private val lifecycleState = MutableStateFlow(CommonLifecycle.State.INITIALIZED)
```

Comment:
I have the same concern about Common Prefix. Can we just name it Lifecycle? We should have common lifecycle and platform lifecycle implementation in the same file